### PR TITLE
Development

### PR DESCRIPTION
**Pull request by newcoder171**

### CODE REVIEW

File: enumerable.rb [hunk 1]
```diff
@@ -57,7 +57,19 @@ def my_none?
...
+
+    end
+  end
+
+end
+
 [1,2,4,5].my_each_with_index { |n, m| puts "#{m}: #{n}"}
-puts [1,2,3,5].my_none? { |n| n>2}
\ No newline at end of file
+puts [1,2,3,5].my_count
```
Comment (heyapricot) [inline enumerable.rb]: - `self.method_name` is redundant
- Prefer the use of `unless` instead of `if !`

File: enumerable.rb [hunk 2]
```diff
@@ -57,7 +57,19 @@ def my_none?
     marker
   end
 
+  def my_count
+    count = 0;
```
Comment (heyapricot) [inline enumerable.rb]: Unnecessary semicolon at the end

**Review by heyapricot — CHANGES_REQUESTED** [review]
# Almost there 🧗🏻‍♀️
1. What happened to`my_map`? It does nothing
2. Please add examples of the use of each of your enumerable methods in a separate file
3. Also, address the comments I've left in the code. 

Thanks 🦊

File: enumerable.rb (hunk 2 above)
Comment (newcoder171) [reply enumerable.rb]: java

**Review by heyapricot — APPROVED** [review]
# Map is working! 🗺
Project fixed! Also, nice first approach on testing. I won't review the test right now. A later assignment requests building tests in Rspec. Once you've done the tests in rspec I'll do the review.
Cya on the next projects 🚗

P.S. add `.idea` to your `.gitignore` file

**Pull request merged**